<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgProductEdit</class>
 <widget class="QDialog" name="dlgProductEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lblName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>290</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblBarcode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Barcode:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledBarcode">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>190</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblPriceIn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Price of buy:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledPriceIn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblPriceOut">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Price of sell:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledPriceOut">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblCount">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Count in stock:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledCount">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="btbButtons">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>156</width>
     <height>30</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::NoButton</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btbCountIncrease">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Increase</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btbCountDecrease">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Decrease</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btbButtons</sender>
   <signal>accepted()</signal>
   <receiver>dlgProductEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btbButtons</sender>
   <signal>rejected()</signal>
   <receiver>dlgProductEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btbCountIncrease</sender>
   <signal>clicked()</signal>
   <receiver>dlgProductEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btbCountDecrease</sender>
   <signal>clicked()</signal>
   <receiver>dlgProductEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
